arcSDinstall.bat

```diff
@@ -0,0 +1,19 @@
+winget install docker.dockerdesktop
+@echo Please open docker desktop and config the proxies.
+@echo If a reboot is needed, do it and open this script again.
+@pause

```

Comment:
If the docker desktop installation doesn't require a reboot (in my case), I think we should restart this bat script before continuing, otherwise the `docker` executable won't be available in the current cmd context.